### ISSUE TITLE
www.iron.io assets are not cached

### ISSUE THREAD

**Issue by odeits** [iron-io/issues#7]
Static resources such as images and script files have their cache control set to no-cache. These assets should be served from a CDN and have long expire times.

**Comment by treeder** [comment]
Good call, we'll be working on this. Starting with changing expiry times. cc @thousandsofthem

**Comment by thousandsofthem** [comment]
``` ruby
config.static_cache_control = "public, max-age=3600" 
```

should help as temporary solution. we must use truly unique file names to achieve longer expire times, e.g. 1 year

**Comment by odeits** [comment]
Possibly helpful resources:
1. http://code.dblock.org/rails-s3-cloudfront-jammit-heroku-100
2. http://code.dblock.org/pushing-assets-to-s3-w-rake-versioning-and-cache-expiration
